src/mono/wasm/runtime/library_mono.js

```diff
@@ -2252,6 +2252,16 @@ var MonoSupportLib = {
 		};
 		debugger;
 	},
+
+	mono_wasm_add_files: function (assembly_ptr, assemnly_len, pdb_ptr, pdb_len) {
+		const assembly_data = new Uint8Array(Module.HEAPU8.buffer, assembly_ptr, assemnly_len);
```

Comment:
Why it's not a good idea to register all the assemblies using this new infrastructure?

Reply:
This will send the assembly as a base64 string, which would be a bigger "download". But also, I would want to do a lot more testing, and checking that moving assembly loading for the proxy, entirely to this technique, works well, and is a good idea. So, I think we should use minimize it's use for now.

Reply:
Okay, I agree that for RC2, we should minimize the use of the new technique. Thanks, Ankit.

Reply:
That seems prudent. I assume .NET 6 is the earlier we'd migrate to this then?

> a lot more testing, and checking that moving assembly loading for the proxy, entirely to this technique, works well, and is a good idea.

Curious: what's your intuition on why this would not be a good idea?